templates/replies-json.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+$post = \get_post();
+
+$page =  \get_query_var( 'page' );
+$args = array(
+	'status'  => 'approve',
+	'number'  => '10',
+    'offset'  => $page,
+    'type'    => 'activitypub',
+	'post_id' => $post->ID,
+    'order'   => 'ASC',
```

Comment:
Again sorry for my unuseful comment: probably here the spaces should be replaced with tabs, in order to indent with tabs

Reply:
running phpcs/phpcbf now ;)